.github/workflows/semver-checks.yml

```diff
@@ -22,15 +22,20 @@ jobs:
         run: cargo binstall cargo-semver-checks --no-confirm
       - name: "Run semver checker script"
         run: ./contrib/check-semver.sh
-      - name: "Add PR label to breaking changes"
-        uses: actions-ecosystem/action-add-labels@v1
+      - name: Save PR number
         if: ${{ hashFiles('semver-break') != '' }}
-        with:
-          labels: "API break"
-      - name: Comment PR
-        uses: thollander/actions-comment-pull-request@v2
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          # check if PR_NUMBER is a number
+          if ! [[ "$PR_NUMBER" =~ ^-?[0-9]+$ ]]; then
```

Comment:
This isn't really needed since the attacker won't attack himself.

Reply:
But I prefer to have it in there. It is not doing anything now since the ENV var is set by Github.
Do you really see that it could be a problem in the near future?

Reply:
It's just a bloat. Not a big deal but I'd remove it if it was entirely my choice.